syft/rekor/query_test.go

```diff
@@ -0,0 +1,76 @@
+package rekor
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/sigstore/rekor/pkg/generated/client"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_getAndVerifySbomFromUuid(t *testing.T) {
```

Comment:
@asraa Need some help with your expertise around rekor inclusion proofs.. would you be able to help review this test? Thanks!